spec/unit/defaults_spec.rb

```diff
@@ -119,6 +119,19 @@
     end
   end
 
+  describe 'manage_internal_file_permissions' do
+    describe 'on windows', :if => Puppet::Util::Platform.windows? do
```

Comment:
The conditional in the tests doesn't match the code. The `Puppet::Util::Platform.windows?` in the tests here is preferred.

Reply:
Will fix depending on comment above about whether to use features or util call.

Reply:
All things being equal, `Util` is preferable if we only care about Windows behavior, since you never mock `Util`.

There was a PR to convert everything that direction that was closed - https://github.com/puppetlabs/puppet/pull/6684

You can see some comments about this at https://github.com/puppetlabs/puppet/pull/6448#discussion_r158360810

Reply:
Updated